### PR TITLE
Update date-fns to 2.14

### PR DESCRIPTION
**Pull request by carlosra85**

### CODE REVIEW

File: projects/systelab-translate/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "Translate",
     "i18n"
   ],
-  "version": "7.0.6",
+  "version": "7.1.0",
```
Comment (alfonsserra) [inline projects/systelab-translate/package.json:10]: Isn't it a breaking change? Should we consider a 8.0.0?

Comment (carlosra85) [reply projects/systelab-translate/package.json:10]: Totally right..I chose minor but major could be better. I will also add the breaking change to readme file

File: projects/systelab-translate/src/lib/date-util/date-util.ts [hunk 2]
```diff
@@ -214,4 +215,81 @@ export class DateUtil {
 				return new Date(auxArray[1] + '/' + auxArray[0] + '/' + auxArray[2]);
 		}
 	}
+
+	private convertSystelabLocaleToDateFnsLocale(locale: string): Locale {
```
Comment (alfonsserra) [inline projects/systelab-translate/src/lib/date-util/date-util.ts:219]: What about using a Map?

Comment (carlosra85) [reply projects/systelab-translate/src/lib/date-util/date-util.ts]: Done

**Pull request merged**